sql/core/src/main/scala/org/apache/spark/sql/streaming/DataStreamWriter.scala

```diff
@@ -379,7 +379,7 @@ final class DataStreamWriter[T] private[sql](ds: Dataset[T]) {
 
   private var outputMode: OutputMode = OutputMode.Append
 
-  private var trigger: Trigger = Trigger.ProcessingTime(0L)
+  private var trigger: Trigger = Trigger.processingTime(0L)
```

Comment:
This was already discussed in my last PR -- we can't change this. However I tried to make the checkstyle config skip checking this method. Is it not working?